tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
```

Comment:
You may want to add tests for:
- Incorrect input (e.g. strings)
- Missing input (e.g. no data files, missing data)
- Negative values
- Only one input value

Reply:
Did not add them now for time constrains, but will later